CMakeLists.txt

```diff
@@ -73,7 +73,7 @@ function(build_ngraph)
         ngraph_set(NGRAPH_UNIT_TEST_ENABLE FALSE)
     endif()
 
-    if(NOT ANDROID)
+    if(NOT (ANDROID OR WINDOWS_STORE))
```

Comment:
Do we have some problems with protobuf for UWP?

Reply:
when we compile for UWP we use cross-compilation, but we don't have a prebuilt Windows package with protoc binary, so, currently, I just disabled ONNX due to this reason.
But it can be enabled later